Formula/osc.rb

```diff
@@ -0,0 +1,67 @@
+class Osc < Formula
+  desc "The Command Line Interface to work with an Open Build Service"
+  homepage "https://github.com/openSUSE/osc"
+  url "https://github.com/openSUSE/osc/archive/0.154.0.tar.gz"
+  sha256 "88daae5b94354e9bcab03523aa7e3d270f69ffeaef8e4a1493bce19757d4699d"
+  head "https://github.com/openSUSE/osc.git"
+
+  bottle :unneeded
+
+  depends_on :python if MacOS.version <= :snow_leopard
+  depends_on "swig" => :build
+  depends_on "openssl" # For M2Crypto
+
+  resource "pycurl" do
+    url "https://dl.bintray.com/pycurl/pycurl/pycurl-7.43.0.tar.gz"
+    sha256 "aa975c19b79b6aa6c0518c0cc2ae33528900478f0b500531dbcdbf05beec584c"
+  end
+
+  resource "urlgrabber" do
+    url "http://urlgrabber.baseurl.org/download/urlgrabber-3.10.1.tar.gz"
+    sha256 "06b13ff8d527dba3aee04069681b2c09c03117592d5485a80ae4b807cdf33476"
+  end
+
+  resource "m2crypto" do
+    url "https://pypi.python.org/packages/packages/source/M/M2Crypto/M2Crypto-0.24.0.tar.gz"
+    sha256 "80a56441a1d2c0cf27e725be7554c92598b938fc8767ee2c71fdbc2fdc055ee8"
+  end
+
+  # Fix path to openssl default certificate
+  patch :DATA
```

Comment:
Is this something we can report upstream & ask them to make a little more flexible?